app/presenters/hyrax/users_presenter.rb

```diff
@@ -0,0 +1,55 @@
+module Hyrax
+  class UsersPresenter
+    attr_reader :query, :authentication_key
+
+    def initialize(query:, authentication_key:)
+      @query = query
+      @authentication_key = authentication_key
+    end
+
+    # @return [Array] an array of Users
+    def users
+      @users = search(query)
+    end
+
+    def user_count
+      ::User.not_batch_or_audit_user.count
```

Comment:
Should this also include the `registered` scope so it doesn't include guest accounts?